python/ray/autoscaler/docker.py

```diff
@@ -15,11 +15,19 @@
 def dockerize_if_needed(config):
     if "docker" not in config:
         return config
+
     docker_image = config["docker"].get("image")
     cname = config["docker"].get("container_name")
     run_options = config["docker"].get("run_options", [])
+
+    head_docker_image = config["docker"].get("head_image", docker_image)
+    head_run_options = config["docker"].get("head_run_options", [])
+
+    worker_docker_image = config["docker"].get("worker_image", docker_image)
+    worker_run_options = config["docker"].get("worker_run_options", [])
+
     ssh_user = config["auth"]["ssh_user"]
-    if not docker_image:
+    if not any([docker_image, head_docker_image, worker_docker_image]):
```

Comment:
what is the expected behavior if `worker_docker_image` is specified and neither `docker_iamge` and `head_docker_image` are specified? (and other way around?)

Reply:
I've changed the logic for this, it will now behave as before